lib/client-assets.php

```diff
@@ -1211,8 +1211,8 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		),
 	);
 
+	// Set a locale specific default font.
 	/*
-	 * Set a locale specific default font.
 	 * Translators: Use this to specify the CSS font family for the default font
```

Comment:
PHPCS currently fails: `Empty line required before block comment`.

The `Set a locale specific default font.` comment is a bit redundant as the translator comment kinda explains that anyway.

I'd simply remove it.

Also, I'd write the translator comment like this:

```php
/* Translators: Use this to specify the CSS font family for the default font. */
```

Reply:
Thanks @swissspidy make complete sense thanks, I've adjusted accordingly.